zeq/hbp/detail/imageRawRGBA8.fbs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+
+namespace zeq.hbp;
+
+table ImageRawRGBA8
```

Comment:
PNG and JPG have their own headers describing the data. Therefore the RGBA8 qualifier is only needed for raw data.